File: module_test/src/main/java/com/gas/test/ui/activity/trans/TransActivity.java
package com.gas.test.ui.activity.trans;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.gas.test.R;

public class TransActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.test_activity_trans);
    }

    @Override
    protected void onPause() {
        overridePendingTransition(0,0);
        super.onPause();
    }

    @Override
    public void onBackPressed() {
        super.onBackPressed();
    }
}

File: module_test/src/main/java/com/gas/test/learn/lesson001_lifecycle/TransActivity.kt
package com.gas.test.learn.lesson001_lifecycle

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.gas.test.R
import com.lib.commonsdk.extension.app.debug

class TransActivity : LifecycleActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.test_activity_trans)
    }
}